### ISSUE TITLE
cli product version number behind tags

### ISSUE THREAD

**Issue by chappjc** [btcsuite/btcwallet#790]
https://github.com/btcsuite/btcwallet/blob/40518d709db8627381a934ce71468c6507e4cfc3/version.go#L19-L21

Before next tag, would be ideal to bump the hard-coded version for the cli app.  It's fairly inconsequential for go consumers, but it's nice when running `btcwallet` to have it reflect the general dev cycle that it was build from.

I had mentioned this before, but wanted to make an issue to track it.  :smile:

**Comment by Roasbeef** [comment]
Good idea, we don't use the direct CLI much ourselves, but we should def start to update it properly each time we do a new tag.
